test/rules/assertions/validTypes.js

```diff
@@ -222,16 +290,6 @@ export default {
           }
       `
     },
-    {
```

Comment:
I think this test case was mistakenly added. It’s identical to an [invalid test case](https://github.com/gajus/eslint-plugin-jsdoc/blob/f5c485da9f91f15da50f88c218a461252ffe5faf/test/rules/assertions/validTypes.js#L108) and they weren’t properly checked before.